sdks/python/oss/tests/pytest/unit/test_preinit_warnings.py

```
@@ -0,0 +1,127 @@
+"""
+Tests that instrumentation methods emit RuntimeWarning when called before ag.init().
+"""
+
+import warnings
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)